src/main/java/uk/nhs/hee/tis/trainee/sync/api/RecordResource.java

```diff
@@ -0,0 +1,27 @@
+package uk.nhs.hee.tis.trainee.sync.api;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.nhs.hee.tis.trainee.sync.dto.RecordDto;
+
+@RestController
+@RequestMapping("/api")
+public class RecordResource {
+
+  private static final Logger log = LoggerFactory.getLogger(RecordResource.class);
```

Comment:
Logger is a constant so should be all upper case "LOG"

Reply:
fixed, thanks!